README.md

```diff
@@ -39,6 +39,7 @@ A dbt profile can be configured to run against Trino using the following configu
 | password  | Password for authentication | Optional (required if `method` is `ldap` or `kerberos`)  | `none` or `abc123` |
 | http_headers | HTTP Headers to send alongside requests to Trino, specified as a yaml dictionary of (header, value) pairs. | Optional |  |
 | http_scheme | The HTTP scheme to use for requests to Trino | Optional (default is `http`, or `https` for `method: kerberos` and `method: ldap`) | `https` or `http`
+| session_properties | Sets Trino session properties used in the connection | Optional | `{"query_max_run_time": "5d"}`
```

Comment:
Please specify multiple properties here to give a clue to the user what the separator for multiple properties is.

Reply:
How do we set an empty value for a session property in the dictionary? `query_max_scan_physical_bytes`  has no value set when executing `SHOW SESSION`

Reply:
We cannot set empty value for `query_max_scan_physical_bytes` at all. 
```
trino> set session query_max_scan_physical_bytes= '';
Query 20210922_091533_03496_ieee5 failed: line 1:1: size is empty
```